applications/registration-react/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+envsubst  < ./server/env.json.template > ./server/env.json
+
+#http://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/
+pm2-runtime start.js
```

Comment:
Is it possible to avoid pm2 completely?

Reply:
vi kan bytte pm2 til nodemon eller noe tilsvsarende. men kanskje vi kan gå rett til nginx?